### PR TITLE
Integration of DB to audio conversion

### PR DESCRIPTION
**Pull request by MirkoFer-622**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -11,10 +11,11 @@ inputFiles.path=storage/inputFiles/
...
 #database:filesdb, table:files
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
-spring.datasource.username=root
-spring.datasource.password=
+spring.datasource.username=demo
+spring.datasource.password=08deseptiembreAT
 #spring.datasource.url=jdbc:mysql://localhost:3306/filesdb?createDatabaseIfNotExist=true&useSSL=false
-spring.datasource.url=jdbc:mysql://localhost:3306/filesdb
+#spring.datasource.url=jdbc:mysql://localhost:3306/filesdb
+spring.datasource.url=jdbc:mysql://52.15.222.115:3306/demo
```
Comment (starsaminf) [inline src/main/resources/application.properties:18]: Please don't submit sensitive data

Comment (samuelloza) [inline src/main/resources/application.properties]: Please don't submit sensitive data.

File: src/main/java/org/fundacionjala/converter/model/AudioModel.java [hunk 2]
```diff
@@ -30,13 +34,10 @@ public String convertAudio(final String params, final String source, final Strin
...
             System.out.println(result.toString());
-            return "Your converted audio is located at: " + target;
-        } catch (java.io.IOException e) {
-            return e.getMessage();
-        } catch (java.lang.InterruptedException e) {
-            return e.getMessage();
+            return target;
+        } catch (java.io.IOException | java.lang.InterruptedException e) {
+            System.out.println(e.getMessage());
+            return "An error occurs when converting file, please try again.";
```
Comment (roht0515) [inline src/main/java/org/fundacionjala/converter/model/AudioModel.java]: An error occurs while converting file, please try again.
can you modify it?

File: src/main/java/org/fundacionjala/converter/controller/AudioController.java [hunk 3]
```diff
@@ -64,13 +76,31 @@ private String copySourceFile(final AudioParameter param) {
             String fileName = param.getFile().getOriginalFilename();
             String source = input + fileName;
             Files.copy(param.getFile().getInputStream(), Paths.get(source), StandardCopyOption.REPLACE_EXISTING);
+            saveFile(source);
             return source;
         } catch (IOException e) {
             System.out.println(e.getMessage());
-            return "Error al cargar el archivo";
+            return "Error when upload file, please try again.";
```
Comment (roht0515) [inline src/main/java/org/fundacionjala/converter/controller/AudioController.java]: Error, while uploading file, please try again
change this message please

File: src/main/resources/application.properties (hunk 1 above)
Comment (MirkoFer-622) [reply src/main/resources/application.properties]: I corrected it at new commit, could you verify please?

File: src/main/java/org/fundacionjala/converter/model/AudioModel.java (hunk 2 above)
Comment (MirkoFer-622) [reply src/main/java/org/fundacionjala/converter/model/AudioModel.java]: I corrected it at new commit, could you verify please?

File: src/main/java/org/fundacionjala/converter/controller/AudioController.java (hunk 3 above)
Comment (MirkoFer-622) [reply src/main/java/org/fundacionjala/converter/controller/AudioController.java]: I corrected it at new commit, could you verify please?

File: src/main/resources/application.properties (hunk 1 above)
Comment (samuelloza) [reply src/main/resources/application.properties]: thks

**Review by samuelloza — APPROVED** [review]

**Review by Laura4lilavati — APPROVED** [review]

**Pull request merged**